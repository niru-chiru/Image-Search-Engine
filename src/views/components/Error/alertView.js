import toastr from "toastr";
import "./toast.css";

toastr.options = {
  closeButton: true,
  debug: false,
  newestOnTop: true,
  progressBar: true,
  positionClass: "toast-top-right",
  preventDuplicates: true,
  onclick: null,
  showDuration: "300",
  hideDuration: "1000",
  timeOut: "5000",
  extendedTimeOut: "1000",
  showEasing: "swing",
  hideEasing: "linear",
  showMethod: "fadeIn",
  hideMethod: "fadeOut",
};

export function success(message) {
  toastr.success(message);
}

export function error(message) {
  toastr.error(message);
}

export function warning(message) {
  toastr.warning(message);
}

export function info(message) {
  toastr.success(message);
}
