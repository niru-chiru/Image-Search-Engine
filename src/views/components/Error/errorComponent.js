import React from "react";

const ErrorComponent = (props) => {
  const { text } = props;
  return (
    <>
      <div className="error-page-section">
        <span className="error-text">{text}</span>
      </div>
    </>
  );
};
export default ErrorComponent;
