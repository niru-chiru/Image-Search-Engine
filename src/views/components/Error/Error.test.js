import { render } from "@testing-library/react";
import Error from "./errorComponent";

const text = "Error while fetiching images!";

it("matches snapshot", () => {
  const { asFragment } = render(<Error text={text} />);
  expect(asFragment()).toMatchSnapshot();
});
