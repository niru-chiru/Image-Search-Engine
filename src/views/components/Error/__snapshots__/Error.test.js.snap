// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="error-page-section"
  >
    <span
      class="error-text"
    >
      Error while fetiching images!
    </span>
  </div>
</DocumentFragment>
`;
